src/middleware/router.coffee

```diff
@@ -22,11 +22,17 @@ os = require 'os'
 domain = "#{os.hostname()}.#{application.name}.appMetrics"
 sdc = new SDC statsdServer
 
-containsMultiplePrimaries = (routes) ->
+
+isRouteEnabled = (route) -> not route.status? or route.status is 'enabled'
+
+exports.numberOfPrimaryRoutes = numberOfPrimaryRoutes = (routes) ->
```

Comment:
@rcrichton I took a look, but ran into issues with the unit tests. Some of them were failing after I added it in, and I traced the problem to the fact that the virtual binding got "lost" at some point. I think it's a problem with how the schema module (i.e. the actual model/channels file) gets loaded when running the tests - I had the same issue with the indexes (e.g. [here](https://github.com/jembi/openhim-core-js/blob/master/test/unit/alertsTest.coffee#L147) we need to reload the indexes even though they're declared at schema level). I'm sure we can get it to work, but it seems a bit clunky (at least for our codebase) to implement atm.